pkg/controller/podgc/gc_controller_test.go

```diff
@@ -72,6 +73,7 @@ func TestGCTerminated(t *testing.T) {
 		deletedPodNames sets.String
 	}{
 		{
+			// case 1
```

Comment:
instead of that, let's introduce a `name` or `description` field and use it as description for the run before.

Reply:
Fixed.